FluentUI.Demo/src/main/assets/dogfood-release-notes.txt

```diff
@@ -1,11 +1,11 @@
-Release Version 0.1.35:
+Release Version 0.1.36:
 
 What's New:
 
 Fluent V2:
 	Fix/Enhancement:
-	        1. Fluent Icon RTL fix
-	Kotlin and Jetpack Compose Upgrade:
-		1.Jetpack compose BOM: 2023.06.01
-		2.Jetpack compose compiler version: 1.4.7 
-		3.Kotlin version: 1.8.21
\ No newline at end of file
+	        1. Fixed the bug that caused multiline text to overlap in v2CCBActivity
+	        2. Removed the developer id, email, and name from config.gradle and passed them as pipeline variables for better security and privacy
```

Comment:
I think this is not required since it is infra related.

Reply:
Done